DDR Feedback

brockfenbert opened the pull request:
This PR isn't quite ideal. There is not a good way to show particle effects on a canvas (how I wrote DDR) without doing you own animation with shaders or messing with camera settings in a way that effects the rest of the scene. 

I made some particle effects for good/great/perfect DDR hits and left them in but they are not used at the moment.

Instead, there is a pop up at the top of the screen with text - this is kind of hard to track when playing DDR, but maybe if we put a note about what the different colors mean it will be ok.

I will look at finding a way to actually implement the particle effects, but it doesn't seem likely.

You can look at it in the DDR_improvements scene, but all the changes should be applied to the DDR prefab

brockfenbert committed:
some ddr improvements

unbun commented:
A few notes that might help improve the look and feel:

- Im not sure if Bebop is the most readable font. I uploaded a font called JazzCreateBubble thats similar but might be a bit more readable
- When I'm playing ddr, my eyes are pretty much glues to the bottom were the catchers are. So maybe the text that comes up could be placed right beneath those
- Something else that might help would be if the grey key catchers changed colors when you clicked them (maybe the same color as the good/great/perfect text that corresponds with your click).
- Also it might help if there's a sound effect when hitting the key

brockfenbert committed:
add in sound effects and move text to bottom

brockfenbert commented:
Pushed changes:

Moved the text to the bottom and switched the font. I think this is much better

Added in sound effects every time you press a key - I don't think this worked out well. I tried the following iterations:
* noise for perfect/great/ok/miss
* noise for pressing down on a key
* different/same noises for each key/effect

All of these iterations made the game feel like there was too much stuff going on in my opinion. I left in the sound effects playing every time someone presses asdf for testing, but I think we can do without this.

rogershannah reviewed (commented):
it looks good, but I think that it's a little fast. maybe there just needs a few more gaps between the letters?
In regards to the sounds:
- I dig the noises for the key presses
- we could probably turn the buzzing down or off during it
- maybe we could introduce a new song/looping sound for during this to spice it up

brockfenbert committed:
disable buzzing while dancing the wasps away
